core/src/main/java/org/mule/api/registry/LifecycleRegistry.java

```diff
@@ -40,4 +40,22 @@
      */
     Object applyLifecycle(Object object, String phase) throws MuleException;
 
+    /**
+     * Look up a single object by name.
+     * <p/>
+     * Because {@link #lookupObject(String)} might
```

Comment:
Isn't it the case (given this comment is in LifecyceRegistry and not Registry) that with lookupObject the returned object will have had lifecycle applied?